### ISSUE TITLE
font wrapper

### ISSUE THREAD

**Issue by B1gG** [basmilius/weather-icons#4]
Hi @basmilius 

I am not familiar with SVGs and Font design, but I was wondering if you can provide a font wrapping around the icons, so it can be referenced easily. I am thinking of a replacement for [Erik's weather icons](https://erikflowers.github.io/weather-icons/), which are good but too many and no colours.

thanks

**Comment by basmilius** [comment]
I'm not familiar with fonts either, but I can take a look at it. Not sure if animations will be supported.

**Comment by B1gG** [comment]
Yeah, I am concerned about the animations but is you can take a look will be really helpful. If I can be of any help just say it 🙂

**Comment by basmilius** [comment]
Update, I think I have something... I'm currently working on monochrome static versions of the icons, those can be turned into a font. There's a node.js package that can do that for me, so I'm positive about this. Colored icons can work too, but I'm not sure yet how, maybe like how emoji webfonts are created..? Animations will not work however 🙂 

Monochrome icons will look something like this:
<table>
<tr>
<td><img alt="clear-day" src="https://raw.githubusercontent.com/basmilius/weather-icons/2.1/design/monochrome/animation-ready/clear-day.svg" width="128"/></td>
<td><img alt="partly-cloudy-day" src="https://raw.githubusercontent.com/basmilius/weather-icons/2.1/design/monochrome/animation-ready/partly-cloudy-day.svg" width="128"/></td>
<td><img alt="overcast-day" src="https://raw.githubusercontent.com/basmilius/weather-icons/2.1/design/monochrome/animation-ready/overcast-day.svg" width="128"/></td>
</tr>
</table>
